core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-category-list/dot-category-field-category-list.component.html

```diff
@@ -3,41 +3,41 @@
     [ngClass]="{ 'no-overflow-x-yet': emptyColumns().length }"
     class="flex-1 category-list__category-list">
     @for (column of categories(); let index = $index; track index) {
-        <!--dynamic columns-->
-        <div #categoryColumn class="category-list__category-column" data-testId="category-column">
-            @for (item of column; track item.inode) {
-                <div
-                    data-testId="category-item"
-                    class="flex align-content-center align-items-center category-list__item"
-                    [ngClass]="{ 'category-list__item--selected': item.checked }"
-                    (click)="itemClicked.emit({ index, item })">
-                    <p-checkbox
-                        [(ngModel)]="itemsSelected"
-                        [value]="item.key"
-                        (onChange)="itemChecked.emit({ selected: $event.checked, item })" />
+    <!--dynamic columns-->
+    <div #categoryColumn class="category-list__category-column" data-testId="category-column">
+        @for (item of column; track item.key) {
+        <div
+            data-testId="category-item"
+            class="flex align-content-center align-items-center category-list__item"
+            [ngClass]="{ 'category-list__item--selected': item.clicked }"
+            (click)="rowClicked.emit({ index, item })">
+            <p-checkbox
+                [(ngModel)]="itemsSelected"
+                [value]="item.key"
+                (onChange)="itemChecked.emit({ selected: $event.checked, item })" />
 
-                    <label
-                        data-testId="category-item-label"
-                        class="flex flex-grow-1 category-list__item-label"
-                        [ngClass]="{ 'cursor-pointer': item.childrenCount > 0 }"
-                        [for]="item.key"
-                        >{{ item.categoryName }}</label
-                    >
+            <label
+                data-testId="category-item-label"
+                class="flex flex-grow-1 category-list__item-label"
+                [ngClass]="{ 'cursor-pointer': item.hasChildren }"
```

Comment:
@oidacra If is a class without much logic, we could use `[class.cursor-pointer]="item.hasChildren"`

Reply:
changed.